mne_bids/write.py

```diff
@@ -310,6 +325,15 @@ def _participants_json(fname, overwrite=False, verbose=True):
     cols['hand'] = {'Description': 'Handedness of the participant',
                     'Levels': {'R': 'right', 'L': 'left', 'A': 'ambidextrous'}}
 
+    # make sure to append any JSON fields added by the user
+    # Note: mne-bids will overwrite age, sex and hand fields by default
```

Comment:
Can you mention this in the docstring of `write_raw_bids()` too? Right now it only mentions that custom values of the TSV file will be retained.